.github/workflows/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] As suggested by the previous reviewer, please kindly add `test.md` to the `.gitignore` file.
- [ ] Kindly move your `.gitignore` file to the root directory because it is wrongly located in it's current folder
- [ ] [OPTIONAL] I kindly suggest that you put a `/` in front of `node_modules` to indicate that it's a folder.

## EXAMPLE
```
.gitignore
`node_modules/`
`test.md`
```

Reply:
I followed the instruction and made the required changes